scripts/sct_analyze_lesion.py

```diff
@@ -232,6 +232,8 @@ def _measure_volume(self, im_data, p_lst, idx):
         printv('  Volume : ' + str(round(vol_tot_cur, 2)) + ' mm^3', self.verbose, type='info')
 
     def _measure_length(self, im_data, p_lst, idx):
+        print len(self.angles)
```

Comment:
Are these print statements required? If so, you can use sct_utils.printv instead.

Reply:
Removed! Is it now correct?